app/src/main/res/values/dimens.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <dimen name="padding_xs">5dp</dimen>
```

Comment:
Note: File-wide comment - your naming is generally okay but think about some more human-readable naming. `tiny`, `small`, `regular`, `big`, `large` often read quite a bit better than `xs`, `sm` and `sm-plus`. You've gone for T-Shirt sizes which is not that bad, really. But it's a matter of personal preference in the end too.

Reply:
Ok, I'll note it for the future as a better option.